pkgs/by-name/it/itools/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, pkgs
+, pkg-config
+, perl
+, libitl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "itools";
+  version = "1.1";
+
+  nativeBuildInputs = [ pkg-config autoconf automake ];
+  buildInputs = [ libitl perl ];
+
+  outputs = [ "out" "man" ];
+
+  src = fetchFromGitHub {
+    owner = "arabeyes-org";
+    repo = pname;
+    rev = version;
+    hash = "sha256-DxTZaq2SlEmy9k7iAdjctpPkk+2rIaF+xEcfXj/ERWw=";
+  };
+
+  preConfigure = ''
+    autoreconf -f -i
+  '';
+
+  meta = with lib; {
```

Comment:
[Also avoid `with lib;`](https://github.com/NixOS/nixpkgs/issues/208242)

Reply:
These things need to be better documented up front in the docs, not just buried in issues that no one new will ever see...

Reply:
forgot to fix this on the last push, pushing again to fix